### PR TITLE
PORTFOLIO: Added Mobile Menu JS Function

### PR DESCRIPTION
**Pull request by peterdtitan**
Used query selectors to implement a mobile menu that also closes when a menu item is clicked. The following files were added/modified:

- script.js was added
- index.html was modified
- style.css was modified
- linters were added for js

Screenshot 2022-06-14 at 6 58 14 PM 1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
                     <li class="nav-item">
                         <a href="#">Contact</a>
                     </li>
-                    <li class="nav-item">
+                    <li id="hidemail" class="nav-item">
                         <a href="#"><img src="./assets/Disabled.png" alt="mail"></a>
                     </li>
```
Comment (tufoinnkuo10) [inline index.html:33]: - [x] links work well great job. please use the same Figma font weight and size, also the line breaks under the link
- your version
Screenshot (121)
- Figma version
Screenshot (116)

File: index.html [hunk 2]
```diff
@@ -28,7 +28,7 @@
                     <li class="nav-item">
                         <a href="#">Contact</a>
                     </li>
-                    <li class="nav-item">
```
Comment (tufoinnkuo10) [inline index.html:31]: - [x] please fix the popup in such a way that when it comes up the user cant see what you are doing in the background.
Screenshot (122)_LI

File: index.html [hunk 3]
```diff
@@ -28,7 +28,7 @@
...
+                    <li id="hidemail" class="nav-item">
                         <a href="#"><img src="./assets/Disabled.png" alt="mail"></a>
```
Comment (tufoinnkuo10) [inline index.html:32]: - [x] please ensure that the close button is with the mobile popup menu and not separated from it. when scrolling on your mobile menu the close button separates from the other links.
your version 👇🏾
Screenshot (123)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @peterdtitan 
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ Menu close icon works perfectly well

Required Changes ♻️
Check the comments under the review

Naah


Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html (hunk 1 above)
Comment (peterdtitan) [reply index.html:33]: Updated

File: index.html (hunk 3 above)
Comment (peterdtitan) [reply index.html:32]: Made Changes to keep the hamburger menu icon fixed.

File: index.html (hunk 2 above)
Comment (peterdtitan) [reply index.html:31]: Fixed popup and background activity

**Comment by peterdtitan** [comment]
Changed appropriate styling to fit the Figma template.

Screenshot 2022-06-15 at 8 29 00 AM

File: script.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  navMenu.classList.toggle('active');
+  logo.classList.toggle('active');
+  navItem.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-item').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+  logo.classList.remove('active');
+}));
```
Comment (sja-thedude) [inline script.js:17]: - [ ] Great job so far, Kindly make sure to add margin to the top and on the right side as shown below.
- [ ] Kindly make sure that the line after all menu options is of the same width as per the Figma design.
- [ ] Kindly make sure to add space after all menu options and before the line as shown.
- [ ] Optional: Kindly follow the same colour for the close button as per the design.

https://user-images.githubusercontent.com/70269507/173751663-a4e17f6a-99e7-41b6-ad4c-59a881e31ff9.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @peterdtitan, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Great job on making the changes asked previously.

 ## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 4 above)
Comment (peterdtitan) [reply script.js:17]: Updated styling. Matched template.

**Comment by peterdtitan** [comment]
Updated the styling to match the Figma template. Added margins and spaced link separators in the menu.

Screenshot 2022-06-15 at 9 37 54 AM

**Review by koachcode — APPROVED** [review]
Hi @peterdtitan , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**